src/main/java/uk/gov/justice/hmpps/datacompliance/repository/jpa/model/retention/RetentionCheckDataDuplicate.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.justice.hmpps.datacompliance.repository.jpa.model.retention;
```

Comment:
This has just been split out into its own class, separate from the image duplicate class